src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -1,30 +1,60 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Classification;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.ScannableItem;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.isNull;
+
 public class EnvelopeMsg implements Msg {
 
+    @JsonProperty("id")
     private final String envelopeId;
 
-    private final byte[] envelope;
+    @JsonProperty("case_ref")
+    private String caseNumber;
 
-    private final boolean testOnly;
+    @JsonProperty("jurisdiction")
+    private String jurisdiction;
 
+    @JsonProperty("classification")
+    private Classification classification;
 
-    public EnvelopeMsg(String envelopeId) {
-        this(envelopeId, false);
-    }
+    @JsonProperty("doc_urls")
+    private List<String> documentUrls;
 
-    public EnvelopeMsg(String envelopeId, boolean testOnly) {
-        this.envelopeId = envelopeId;
+    @JsonIgnore
+    private final byte[] envelope;
+
+    @JsonIgnore
+    private final boolean testOnly;
```

Comment:
I think this field is not required any more. Please check `bulk-scan-orchestrator`

Reply:
I think that field is still needed indeed. Better check bulk-scan-orchestrator.

Reply:
As far I know it was removed so yes better check bulk-scan-orchestrator 😉 https://github.com/hmcts/bulk-scan-orchestrator/blob/master/src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/EnvelopeEventProcessor.java

Reply:
It was removed in its current implementation, as we are changing the messages substantially, but I believe there is still a need for a test message so that during tests, cases are not created in CCD. Rafal is taking care of this in the orchestrator so he's probably the best person to answer that question

Reply:
Tests are executed on AAT so we would want them to get created on AAT. Why do you think we should not create cases ?

Reply:
Because we will want to avoid hitting ccd in some test cases? Also this way we can implement syntethic transactions to monitor that the processor and orchestrator can talk while in prod (without creating cases in ccd prod).